assets/src/edit-story/components/canvas/rightClickMenu.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { AnimatedContextMenu } from '../../../design-system';
+import { useRightClickMenu } from '../../app/rightClickMenu';
+import { RightClickMenuContainer, RightClickPageZone } from './layout';
+
+const RightClickMenu = () => {
+  const {
+    isMenuOpen,
+    menuItems: rightClickMenuItems,
+    menuPosition,
+    onCloseMenu,
+    rightClickAreaRef,
+  } = useRightClickMenu();
+
+  return (
+    <RightClickPageZone ref={rightClickAreaRef}>
```

Comment:
Maybe this isn't the right time for this, but I think this `ref` should go on the `FullBleedContainer` so that the clicking is contained to the canvas and doesn't block the rest of the editor.

Reply:
Yea current implementation exhibits this behavior. I believe the `rightClickAreaRef` is just used for positioning. But I agree I think this can be mitigated by only attaching the 'contextmenu' event to a top level component occupying the space we want this functionality to apply to.
https://user-images.githubusercontent.com/35983235/121724170-1d9e5200-caad-11eb-9813-939a60e6940d.mp4